### ISSUE TITLE
Incorrect offense report: Use attr_reader to define trivial reader methods

### ISSUE THREAD

**Issue by argos83** [bbatsov/rubocop#2140]
Since Rubocop v0.33.0, I started getting the following offense which I don't think is correct:

Test file:
```ruby
class DummyClass
  def initialize
    @failed = false
  end

  def failed?
    @failed
  end
end

```

Rubocop output:
```
Inspecting 1 file

C

Offenses:

    temp.rb:7:3: C: Use attr_reader to define trivial reader methods.
def failed?
^^^

    1 file inspected, 1 offense detected
```

I wish to have a `failed?` with question mark method (as it returns a boolean), and I can't define the attribute with a question mark. So adding an `attr_reader` doesn't help. (I workaround it by renaming the instance var to `@failure`

Thanks for this great library!

**Comment by mattjmcnaughton** [comment]
Hi @argos83!

I did some poking around in the source and it looks like this behavior was introduced on purpose in this (commit)[https://github.com/bbatsov/rubocop/commit/e771f4b4681b2bfa81f48b04795a5319dd0543f1].

However, you can turn the behavior off by the following code to your `.rubocop.yml`.
```
Style/TrivialAccessors:
  AllowPredicates: true
```

or by enclosing the method in
```
# rubocop:disable TrivialAccessors
YOUR_METHOD...
# rubocop:enable TrivialAccessors
```

Hope that helps!

**Comment by murphyslaw** [comment]
:+1:

**Comment by bbatsov** [comment]
Yep, that's not a bug. Perhaps we should change the default, but things are working as expected.

**Comment by argos83** [comment]
Thanks for the input!

**Comment by bolshakov** [comment]
Let's change defaults!
